content/html/concepts/tags/terms/blink/blink.md

```diff
@@ -0,0 +1,25 @@
+---
+Title: '<blink>'
+Description: 'Obsolete tag that originally caused text to flash on and off. It is now deprecated and non-standard.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+  - 'Style'
+  - 'Obsolete'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<blink>` tag is an obsolete tag that was used historically to make text flash on and off. It is non-standard, depreciated, and not part of the HTML specifications.
+
+Using this element is a bad design practice and is discouraged by accessibility standards. The CSS specification allows browsers to ignore this tag.
```

Comment:
```suggestion
The `<blink>` tag was used historically used to make text flash on and off. However, it is now considered to be obsolete and non-standard. It is now, or in the middle of being, depreciated from most browsers is not a part of the HTML specifications.

Using this tag is a bad design practice and is heavily discouraged by accessibility standards. The CSS specification allows browsers to ignore this tag.
```

I feel like there are some things that we could/should link to in these paragraphs:

- "HTML specifications" (are we talking about [this](https://html.spec.whatwg.org/)?)
- "accessibility standards" (are we talking about [this](https://www.w3.org/WAI/standards-guidelines/wcag/)?)
- "CSS specification" (most recent snapshot from 2021 appears be [this](https://www.w3.org/TR/css-2021/))